config/hocon-mp/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.config</groupId>
+        <artifactId>helidon-config-project</artifactId>
+        <version>2.5.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>helidon-config-hocon-mp</artifactId>
+    <name>Helidon Config HOCON MP</name>
+
+    <description>
+        HOCON support for Helidon MP.
+    </description>
+
+    <properties>
+        <spotbugs.exclude>etc/spotbugs/exclude.xml</spotbugs.exclude>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
This dependency is not used. If this is to add native image support, then we should modify our module structure.

Reply:
The way I tested config profile is by using `-Dconfig.profile` and it only works with the addition of the above dependency. Digging a little bit, and this was a mystery a few days ago, I think I seem to understand how it works now. MpConfigProviderResolver.buildConfig() has this code: 
```
        // MP Meta Configuration
        Optional<io.helidon.config.Config> meta = MpMetaConfig.metaConfig();

        if (meta.isEmpty()){
            meta = MetaConfig.metaConfig();

            if (meta.isPresent()) {
                builder.metaConfig(meta.get());
                LOGGER.warning("You are using Helidon SE meta configuration in a Helidon MP application. Some features "
                                       + "work differently, such as environment variable resolving, and mutability");
            }
        } else {
            builder.mpMetaConfig(meta.get());
        }
```

and if MetaConfig.metaConfig() is null (i.e. no mp-meta-config.profile/yaml) then it will run  `meta = MetaConfig.metaConfig()` which eventually will result to getting the config source of config profile, like `application-<profile>.conf/json`.

Reply:
This is Helidon SE config feature, not Helidon MP. It is in the warning directly in the code. 
If people want to use Helidon SE meta configuration with Helidon MP and with Hocon, let them add this dependency on their own.

Reply:
removed